app/src/main/java/com/codesoom/assignment/controllers/TaskController.java

```diff
@@ -0,0 +1,86 @@
+package com.codesoom.assignment.controllers;
+
+
+import com.codesoom.assignment.models.Task;
+import com.codesoom.assignment.repository.TaskService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+/**
+ * The class is responsible for mapping /tasks requests
+ */
+@RestController
+@RequestMapping("/tasks")
+public class TaskController {
+    private final TaskService taskService;
+
+    public TaskController(TaskService taskService) {
+        this.taskService = taskService;
+    }
+
+    /**
+     * Returns all lists of tasks from the database
+     * @return the List of tasks
+     */
+    @GetMapping
+    public List<Task> list() {
+        return taskService.findAllTasks();
+    }
+
+    /**
+     *
+     * Returns ResponseEntity with the found task
+     * @param id from URI parameter
+     * @return ResponseEntity with HttpStatus.OK; otherwise, returns ResponseEntity with HttpStatus.NOTFOUND
```

Comment:
간단하게 표현해보면 어떨까요? `@return`의 내용은 핵심적인 내용을 짧게 표현해 보세요.

```suggestion
     * @return 조회 결과
```